external-crates/move/crates/move-package/src/compilation/build_plan.rs

```diff
@@ -144,4 +209,20 @@ impl BuildPlan {
         }
         Ok(())
     }
+
+    pub fn root_package_path(&self) -> PathBuf {
+        self.resolution_graph.package_table[&self.root]
+            .package_path
+            .clone()
+    }
+
+    pub fn record_package_edition(&self, edition: Edition) -> anyhow::Result<()> {
```

Comment:
Will these comments and the like that were in the original TOML file? Just slightly worried here that it may strip those out.

Reply:
The library quite kindly preserves comments, but I did add a test to ensure this.

Reply:
Amazing! That's great news 🙂